k8s/upgrade/src/upgrade_resources/upgrade.rs

```diff
@@ -7,15 +7,23 @@ use crate::{
         UPGRADE_JOB_IMAGE_NAME, UPGRADE_JOB_IMAGE_REPO, UPGRADE_JOB_NAME_SUFFIX,
         UPGRADE_JOB_SERVICEACCOUNT_NAME_SUFFIX,
     },
-    error::Error,
+    error::{
+        ClusterRoleBindingCreate, ClusterRoleBindingDelete, ClusterRoleCreate, ClusterRoleDelete,
```

Comment:
wow this is huge, I suggest use it with the module, example: `error::ClusterRoleBindingCreate`

Reply:
done